docs/Clearwater_SNMP_Statistics.md

```diff
@@ -111,6 +111,12 @@ Homestead nodes provide the following statistics:
 * The number of incoming requests, indexed by time period.
 * The number of requests rejected due to overload, indexed by time period.
 * The total number of Diameter requests with an invalid Destination-Realm or invalid Destination-Host, indexed by time period.
+* The number of Multimedia-Authorization-Requests sent over the Cx interface, indexed by time period and result code.
```

Comment:
The phrasing here isn't very clear that it's not primarily about counting requests sent, but about tracking response codes. I don't have any really good ideas on how to make it clearer - maybe "The number of Multimedia-Authorization-Answers with a given result code received over the Cx interface..."?